core/src/main/python/wlsdeploy/util/cla_helper.py

```diff
@@ -305,6 +305,7 @@ def merge_model_files(model_file_value, variable_map=None):
     It may be a single file, or a comma-separated list of files.
     :param variable_map: variables to be used for name resolution, or None
     :param model_file_value: the value specified as a command argument
+    ;param model_context:  model context
```

Comment:
looks like a semi-colon here instead of a colon

Reply:
model_context isn't an argument any more

Reply:
removed